WordPress/Classes/Utility/PushNotificationsManager.swift

```diff
@@ -4,8 +4,6 @@ import UserNotifications
 import CocoaLumberjack
 import UserNotifications
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-iOS/blob/bcfb96f7b5bf2a8e02d0b5434926cab4a59d0f54/WordPress/Classes/Utility/PushNotificationsManager.swift#L5"></span>Imports should be unique<br />
<code>duplicate_imports</code> <code>PushNotificationsManager.swift:5</code></td>
    </tr>
  </tbody>
</table>